src/config/parse.c

```diff
@@ -1424,9 +1440,6 @@ configParse(const Storage *storage, unsigned int argListSize, const char *argLis
                         optionArg = strNewZ(argList[++argListIdx]);
                     }
                 }
-                // Else error if an argument was found with the option
```

Comment:
What about something like `--reset-pg1-host=xxx`? Looks like we need a new test for this condition now that the old one is not valid.

Reply:
restored check and added test in commit 473e5597.